packages/react-html/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
```

Comment:
It looks like the changelog process changed, but I can't find the new documentation about it.

Reply:
We're using `changesets` now, with some details here: https://github.com/Shopify/quilt/blob/main/documentation/release-and-deploy.md#creating-releases
We should improve our docs to make this easier to discover

tl;dr; is run `yarn changeset` and follow the prompts, then commit the changeset file. The rest will be handled by github actions.